### PR TITLE
BZ 2011:  oic.wk.mnon.raml

### PR DESCRIPTION
**Pull request by WAvdBeek**

### CODE REVIEW

File: oic.wk.mnon.raml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#%RAML 0.8
+title: Maintenance
+version: v1-20171220
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2016, 2017 Open Connectivity Foundation, Inc. All rights reserved.
```
Comment (WAvdBeek) [inline oic.wk.mnon.raml]: add 2018

File: oic.wk.nmon.raml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        body:
+          application/json:
+            schema: NMON
+            example: |
+              {
+                "rt":   ["oic.wk.nmon"],
+                "ianaifType":   71,
+                "reset":   false,
+                "col" : false,
+                "time" : "",
```
Comment (mkettrick) [inline oic.wk.nmon.raml]: This value of "time" will not pass schema validation.

File: oic.wk.nmon.raml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      200:
+        body:
+          application/json:
+            schema: NMON
+            example: |
+              {
+                "rt":   ["oic.wk.nmon"],
+                "ianaifType":   71,
+                "reset":   false,
+                "col" : false,
```
Comment (mkettrick) [inline oic.wk.nmon.raml]: Why is the value of "col" FALSE when the UPDATE request changed it to TRUE?

Comment (WAvdBeek) [reply oic.wk.nmon.raml]: see state diagram... this is resetting existing values..

Comment (WAvdBeek) [reply oic.wk.nmon.raml]: misread the comment, fixed..

File: oic.wk.nmon.raml (hunk 2 above)
Comment (mkettrick) [reply oic.wk.nmon.raml]: Any thoughts on fixing this issue?

File: oic.wk.nmon.raml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        body:
+          application/json:
+            schema: NMON
+            example: |
+              {
+                "rt":   ["oic.wk.nmon"],
+                "ianaifType":   71,
+                "reset":   false,
+                "col" : true,
+                "time" : "",
```
Comment (mkettrick) [inline oic.wk.nmon.raml]: Not sure what happened to my comment but this needs to be updated to reflect a valid date-time.

**Comment by WAvdBeek** [comment]
replaced by:
https://github.com/openconnectivityfoundation/core/pull/118

**Pull request closed without merging**